src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -8,7 +8,11 @@ Release History
 * appservice plan, webapp & function app updated to use latest python websites SDK version
 * az webapp config set supports --fts-state property. Also added support fot az functionapp config set & show
 * webapp: add support for bring your own storage
+* az functionapp create supports creating a linux consumption plan type with a specific runtime
 * webapp: add support for listing and restoring deleted apps
+* az functionapp create supports creating a linux consumption plan type with a specific runtime
+* az appserviceplan create now supports creating Windows Containers App Service Plans - Preview
+* az webapp create create now supports creating and managing Windows Containers Apps - Preview
```

Comment:
"create" is used twice here.
i don't see any history changes referring to `appservice plan update` now allowing for a new argument.
Are there code changes for `webapp create` and `functionapp create`?

Reply:
There are ineedchanges to `web app create` for example, [this.](https://github.com/Azure/azure-cli/pull/7390/files#diff-35983163c0f2f417dda0c571d4a4538dR91)

`az functionapp create` should not be part of this PR. Will remove it.

We are not changing `appservice plan update` because changing the appservice plan "type" (for example from "Linux" to "Windows" ) is not supported. Once an appservice plan has been created with a certain "type" it cannot be changed.

Reply:
Thanks for the changes and explanation.
i was mistaken about `appservice plan update`. I thought that it was the command receiving the new `windows_fx_version` parameter.  However, this is a new parameter for the commands `az webapp config set` and `az functionapp config set`.  Shouldn't there be an update to history.rst explaining that this new parameter can be used for these commands? 

Or is this overkill, @yugangw-msft?

Reply:
I think aside from this last request, this generally looks good to me. Although, @yugangw-msft will also give a look through the PR.

Reply:
@adewaleo this parameter is intended only for `webapp`s, not `functionapp`s. Although the underlying API won't fail, setting this parameter in a function app doesn't have any effect. I will update the parameter help and updated the end-to-end test to test az webapp config set as suggested.

Reply:
I see. Thanks for incorporating the changes, @vinisoto 

I would add though that If the parameter is unnecessary for the functionapp command, won't it be better / clearer not to expose it to users at all? One possible way to do this is to use an if statement within the argument context based on the value of scope. 
This way the parameter only shows up for relevant commands and the help text can be more concise / relevant to the command.

What are your thoughts on this?